resources/lang/en/events/lead_ceremony_sc.json

```diff
@@ -465,7 +444,7 @@
       "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy"],
       "life_giving": [
         {
-          "text": "r_c advances forward, chuckling and gently nudging m_c as {PRONOUN/r_c/subject} {VERB/r_c/urge/urges} {PRONOUN/m_c/object} to fix {PRONOUN/m_c/poss} meek posture to one that emanates confidence and commands respect. Only then does r_c grant a life for [virtue], hoping it will remind m_c of {PRONOUN/m_c/poss} worth and the faith c_n has in {PRONOUN/m_c/object}.",
+          "text": "r_c advances, chuckling and gently nudging m_c as {PRONOUN/r_c/subject} {VERB/r_c/urge/urges} {PRONOUN/m_c/object} to correct {PRONOUN/m_c/poss} meek posture and instead adopt one that emanates confidence and commands respect. Only then does r_c grant a life for [virtue], hoping it will remind m_c of {PRONOUN/m_c/poss} worth and the faith c_n has in {PRONOUN/m_c/object}.",
```

Comment:
```suggestion
          "text": "r_c advances, gently nudging m_c to correct {PRONOUN/m_c/poss} posture to one that emanates confidence and commands respect. Only then does r_c grant a life for [virtue], hoping it will remind m_c of {PRONOUN/m_c/poss} worth and the faith c_n has in {PRONOUN/m_c/object}.",
```